Tiltfile

```diff
@@ -0,0 +1,22 @@
+# Eraser Tiltfile.
+
+print("""
+-----------------------------------------------------------------
+ERASER(📒 ✏️) Assumes docker installed, a kind cluster instantiated and kubectl in the right context.
+-----------------------------------------------------------------
+""".strip())
+
+docker_build('eraser-build', context='.')
+
+k8s_yaml('./deploy/eraser.yaml')
+# add further deployments above. Reference sample tilt file. or https://docs.tilt.dev/install.html
+
+local_resource('Kubectl get pods',cmd='kubectl get pods -A')
+local_resource('Kubectl get imageList',cmd='kubectl get ImageList -A')
+local_resource('Kubectl describe imageList',cmd='kubectl describe ImageList imagelist -A')
```

Comment:
not familiar with these, what are the local resource for?

Reply:
They are simply `kubectl` commands. These are the commands in the Getting started, I included as samples of commands that can be utilized while using a tilt file.